### PR TITLE
add a few newer extensions to the reports

### PR DESCRIPTION
**Pull request by nicolethoen**

### CODE REVIEW

File: src/static-analysis/getPackageStats.js [hunk 1]
```diff
@@ -134,9 +140,9 @@ function getPackageStats(repoPath, repoName, url) {
         };
 
       //collect row for output object
-       if(pp || pq || prcve ||prc || prce || prcon || prcore || pricon || predit || prlogv || pnext || prstyle || prtable || prtoken || prtop || pruf || prvirt || pf || pfr || pfre != null){
+       if(pp || pq || prcve ||prc || prce || prcon || prcore || pricon || predit || prlogv || prstyle || prtable || prtoken || prtop || pruf || prvirt || pf || pfr || prchat || prdata || prcomp || pfre != null){
```
Comment (pilhuhn) [inline src/static-analysis/getPackageStats.js:143]: Is it ok that you insert the new items in a place that is different from where you removed `pnext` ?

It is only with in a list of or-ed together items, but for consistency moving those to the other place may be better.

File: to_xls.py [hunk 2]
```diff
@@ -28,14 +28,14 @@
 # Helper method to clean up cell values by
 #  turning 'undefined' and 'null' into empty cell
 def clean_undefined(x) -> str:
-    if "undefined" in x:
-        return ''
-    if "null" in x:
-        return ''
+    if isinstance(x, (str, list, tuple, set, dict)):
```
Comment (pilhuhn) [inline to_xls.py:31]: Did you see the original code crashing? 
if so, we should perhaps also handle the `not ( isinstance ... ` case (?)

File: src/static-analysis/getPackageStats.js [hunk 3]
```diff
@@ -113,18 +116,6 @@ function getPackageStats(repoPath, repoName, url) {
...
-        else if(key == 'patternfly'){
-          pf = courses[key];
-        }
-        else if(key == 'patternfly-react'){
-          pfr = courses[key];
-        }
-        else if(key == 'patternfly-react-extensions'){
-          pfre = courses[key];
-        }
-        else if(key == 'react'){
```
[the remark this answers is not in the document]
Comment (nicolethoen) [reply src/static-analysis/getPackageStats.js:119]: I moved it up to line 107 so that they'd all be in the same order

File: to_xls.py (hunk 2 above)
Comment (nicolethoen) [reply to_xls.py:31]: I think upon further inspection I can fix the error i was seeing more simply. I'll try it out

File: repos_2023.json [hunk 4]
```diff
@@ -188,6 +196,14 @@
     {
       "git": "https://github.com/nmstate/nmstate-console-plugin",
       "name": "Nmstate-plugin"
+    },
+    {
+      "git": "https://github.com/instructlab/ui",
```
Comment (pilhuhn) [inline repos_2023.json:201]: Does that make sense? Instructlab (UI) did not exist in 2023

**Comment by nicolethoen** [comment]
@pilhuhn I'd love to merge this so subsequent runs of the analytics include the cleaned up repo list :) I can also open a follow up issue if needed for small tweaks.

**Pull request merged**